airflow/www/templates/airflow/dag.html

```diff
@@ -36,7 +36,7 @@
   <meta name="extra_links_url" content="{{ url_for('Airflow.extra_links') }}">
   <meta name="paused_url" content="{{ url_for('Airflow.paused') }}">
   <meta name="tree_data" content="{{ url_for('Airflow.tree_data') }}">
-  <meta name="is_paused" content="{{ dag.is_paused }}">
+  <meta name="is_paused" content="{{ dag.get_is_paused() }}">
```

Comment:
I believe there are a few more places in `/www` that we use `is_paused`

Reply:
All replaced now! (Actually only a handful across two files.)